File: ReptiCare/Defecation+CoreDataProperties.m
//
//  Defecation+CoreDataProperties.m
//  ReptiCare
//
//  Created by Kevin De Koninck on 24/09/2016.
//  Copyright © 2016 Kevin De Koninck. All rights reserved.
//

#import "Defecation+CoreDataProperties.h"

@implementation Defecation (CoreDataProperties)

+ (NSFetchRequest<Defecation *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Defecation"];
}

@dynamic date;
@dynamic kindOfDefecation_;
@dynamic note;
@dynamic uniqueID;
@dynamic reptile;

@end

File: ReptiCare/Defecation+CoreDataProperties.swift
//
//  Defecation+CoreDataProperties.swift
//  ReptiCare
//
//  Created by Kevin De Koninck on 24/09/2016.
//  Copyright © 2016 Kevin De Koninck. All rights reserved.
//

import Foundation
import CoreData 

extension Defecation {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Defecation> {
        return NSFetchRequest<Defecation>(entityName: "Defecation");
    }

    @NSManaged public var date: NSDate?
    @NSManaged public var kindOfDefecation_: NSNumber?
    @NSManaged public var note: String?
    @NSManaged public var uniqueID: NSNumber?
    @NSManaged public var reptile: Reptile?

}
